all-in-one/BUILDING-TESTING.md

```diff
@@ -63,6 +72,11 @@ most cases this is not necessary as the `_GIT_BRANCH` build arguments support bo
 
 ## Testing
 
-The container can be tested using end-to-end Cypress tests.
-
+The container can be tested using end-to-end Cypress tests:
 More information on testing can be found in [end-to-end-tests.](../end-to-end-tests)
+
+After you successfully create a new docker image, to pull the newly created image run:
```

Comment:
```suggestion

After you successfully create a new docker image, to run the newly created image execute:
```

Reply:
I like this suggestion, small change that phrases the sentence better